src/CompileTime.hx

```diff
@@ -64,6 +68,17 @@ class CompileTime
         return toExpr(obj);
     }
 
+
+    macro public static function parseYamlFile(path:String) {
```

Comment:
Can you wrap this (and the imports) in a `#if yaml` conditional macro?

It would be good to have this if the yaml haxelib is used, but to just silently ignore it otherwise.